.eslintrc

```diff
@@ -127,6 +127,7 @@
     "react/jsx-wrap-multilines": "off",
     "react/jsx-one-expression-per-line": "warn",
     "react/jsx-boolean-value": "off",
+    "react/require-default-props": "off",
```

Comment:
What is the idea behind this?

Reply:
That rule has not been active before, but in updated eslint it is and it shows a lot of errors in the interfaces. It would want all the optional props to have a default value. I think that is not something we want to have. 
```
export interface Something {
   text?: string
}
```
in this example there should be some default value for `text`